### ISSUE TITLE
Copy and Paste does not work as expected

### ISSUE THREAD

**Issue by matthewhorridge** [protegeproject/snap-sparql-query#23]
Copying of selections does not work

**Comment by csnyulas** [comment]
Indeed, paste from outside source works, but copy does not, which makes reuse of (parts of) already typed query impossible. Would be really useful to have the copy working.

BTW, I just realized that "cut" (CTRL+X) does work, so one could simulate a "copy" with a cut followed by a paste.

**Comment by csnyulas** [comment]
BTW, The context menu option "Copy as Rich Text" is not working either.

**Comment by yasenstar** [comment]
I'm in Protege 5.6.2, this copy is not working from Snap SPARQL plug-in still, so I cannot copy the statement out for documentation purpose. Any plan to fix it?
